dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -166,7 +166,13 @@ requireRequestUriRegistration=Require request URI registration
 requireRequestUriRegistration.desc=Indicate by true or false whether require request URI registration is supported.
 
 customClaims=Extra claims of ID token
-customClaims.desc=The extra claims to be put in the payloads of the ID Token, in addition to the default realmName, uniqueSecurityName and groupIds claims.
+customClaims.desc=The extra claims to be put in the payloads of the ID token, in addition to the default realmName, uniqueSecurityName and groupIds claims.
```

Comment:
Add a comma after uniqueSecurityName, like so:
to the default realmName, uniqueSecurityName, and groupIds claims.